pyface/tasks/action/task_window_toggle_group.py

```diff
@@ -101,7 +101,7 @@ def _get_items(self):
             items.append(ActionItem(action=action))
         return items
 
-    def _rebuild(self):
+    def _rebuild(self, _=None):
```

Comment:
```suggestion
    def _rebuild(self, event):
```

but the same comment here i.e. why `_rebuild` isn't simply decorated.